webapp-user/.gitignore

```diff
@@ -0,0 +1,19 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
There are extra things here that are happening because of the way that create-react-app is being used - particularly related to css files. The css files need to be ignored as they are a part of the build process. So the .gitignore is a part of the create-react-app config.